### ISSUE TITLE
Documenting how to set "development team"

### ISSUE THREAD

**Issue by LinusU** [yonaskolb/XcodeGen#262]
**Question/Feature request:**

Is it possible to set the "development team" used for code signing from the `project.yml` file?

**Background:**

I'm trying to set up a test iOS app where I will not use Xcode at all to develop. My idea is to have something like this:

```makefile
.PHONY: run build

SOURCES := $(shell find Sources -name '*.swift')

Name.xcodeproj: project.yml
    mint run yonaskolb/xcodegen

build/Release-iphoneos/Name.app: Name.xcodeproj $(SOURCES)
    xcodebuild

run: build/Release-iphoneos/Name.app
    ios-deploy --bundle build/Release-iphoneos/Name.app --justlaunch
```

Now I'm stuck on:

```text
Check dependencies
Code Signing Error: Signing for "Name" requires a development team. Select a development team in the project editor.
Code Signing Error: Code signing is required for product type 'Application' in SDK 'iOS 11.2'

** BUILD FAILED **
```

**Comment by LinusU** [comment]
Turns out that this was super easy, I figured it out after manually diffing the xcodeproj file after setting the development team.

The trick is:

```yaml
# ...
targets:
  Name:
    # ...
    settings:
      DEVELOPMENT_TEAM: XXXXXXXXX
```

-----

Would it be possible to add this to the readme? or maybe we could create an FAQ?

For reference, I tried searching for "team" and "sign" in the issues and this document: https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md. But I didn't find anything.

**Comment by toshi0383** [comment]
`DEVELOPMENT_TEAM` isn't special in project's spec context, but maybe it worth having such documentation for beginners.
Maybe not readme or `ProjectSpec.md`, but in different page such as `Docs/GettingStarted.md` or `Docs/FAQ.md`.
@LinusU PullRequest is welcomed👍

By the way, you can define `DEVELOPMENT_TEAM` in multiple way. You might want to document these too.
- by defining in xcconfig file
- by passing as environment variable. e.g. `DEVELOPMENT_TEAM=XXXXXXX xcodebuild archive ...`

**Comment by LinusU** [comment]
> DEVELOPMENT_TEAM isn't special in project's spec context, but maybe it worth having such documentation for beginners.

Yeah, exactly my thoughts 👍 

Opening a PR shortly...

**Comment by LinusU** [comment]
(closed by #267)
